bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CCombo.java

```diff
@@ -2025,4 +2025,13 @@ public boolean traverse(int event){
 	}
 	return super.traverse(event);
 }
+/**
+ * @since 3.126
```

Comment:
some javadoc remarks:
- I think the since version must be 3.127
- the javadoc should contain a description what the method is doing (passing all widgets managed by the ccombo to the specified consumer)
- I think the method name should be more explanatory, but currently dont have a good idea myself

Reply:
and what was the argument of static vs. not static again? @fedejeanne @HeikoKlare

Reply:
> * I think the method name should be more explanatory, but currently dont have a good idea myself

We may align with the name for `StyledText` used in #1294, which is `updateAndRefreshCarets`, so maybe more general `updateAndRefreshChildren`?
In addition, the parameter `consumer` should have a more specific name like `childUpdater`.

> and what was the argument of static vs. not static again?

Non-static would mean that this becomes public API. I would propose to add a `@noreference` and keep it static to not have it as part of public API. Then you can also remove the `@since` tag.